runtime/src/main/java/uk/ac/manchester/tornado/runtime/tasks/TornadoTaskSchedule.java

```diff
@@ -626,6 +640,12 @@ private AbstractTaskGraph analyzeSkeletonAndRun() {
 
     @Override
     public AbstractTaskGraph schedule() {
+
+        if (getTornadoRuntime().getNumDrivers() < 1 || getTornadoRuntime().getDriver(0).getNumPlatforms() < 1) {
+            runSequential();
```

Comment:
Introduce a warning that the code was "deoptimize" and TornadoVM will execute the Java sequential code.